packages/projects/src/constants.js

```diff
@@ -13,6 +14,7 @@ const components = {
   li: (props) => <li {...props} className="my-2" />,
   p: (props) => <p {...props} className="my-4" />,
   hr: (props) => <hr {...props} className="my-4" />,
+  Link: (props) => <Link {...props} to={props.to}>{props.children}</Link>,
```

Comment:
Nice. An alternative thought I had was to actually inspect the href property of an a component and use Link for relative URLs just so the user doesn't have to remember to use something special. Next iteration.

Reply:
While we're on the topic. I was thinking to just use dummy components while editing for the embedded components like figma etc. to not disrupt the editing and steal the focus. The dummy would just render some sort of outline with some standard width x height and then on the view page we would use the actual components.

let me know if this makes sense.

Reply:
@simonkernel I thought about inspecting the `<a>` as well, but I think many users would just try to put the whole URL anyway so either we'd still have to tell them to use relative, or we would have to inspect for "is this domain the same as our domain?" and I didn't feel like doing that right now 😄 let's see how often people actually want to link to another adventure.

Dummy components sounds reasonable.. but how does the component know whether it's in preview vs. real view? 🤔